students/chylak_malgorzata/lesson_06_dictionaries_tuples_and_sets/logger_wrapper.py

```diff
@@ -0,0 +1,21 @@
+from args_inspector import inspect_args
+
+
+def sum_all(*args):
+    result = 0
+    for arg in args:
+        result = result + arg
+    return result
+
+
+def logger_wrapper(foo, *args, **kwargs):
```

Comment:
An argument ```foo``` shadows ```foo``` function. I propose to change ```foo``` function name to any else 😄